src/ServiceFabric/ServiceFabric/Models/ManagedClusters/PSManagedNodeType.cs

```diff
@@ -24,22 +24,22 @@ public PSManagedNodeType(NodeType nodeType)
                    type: nodeType.Type,
                    tags: nodeType.Tags,
                    isPrimary: nodeType.IsPrimary,
-                   vmInstanceCount: nodeType.VmInstanceCount,
-                   dataDiskSizeGB: nodeType.DataDiskSizeGB,
+                   vmInstanceCount: nodeType.VMInstanceCount,
```

Comment:
Seems this property is changing, which might cause a breaking change. Could you please help confirm whether this will affect cx?
```
"Az.ServiceFabric","Microsoft.Azure.Commands.ServiceFabric.Commands.AddAzureRmServiceFabricClientCertificate","Add-AzServiceFabricClientCertificate","0","3010","The property 'VmInstanceCount' of type 'Microsoft.Azure.Management.ServiceFabric.Models.NodeTypeDescription' has been removed.","Add the property 'VmInstanceCount' back to type 'Microsoft.Azure.Management.ServiceFabric.Models.NodeTypeDescription'."
```

Reply:
should not affect how the command is ran or the output. The property nodeType.VMInstanceCount is the internal property which is assigned to vmInstanceCount which is what is actually returned in output